components/driving_with_remote_controller.mdx

```diff
@@ -0,0 +1,40 @@
+:::caution
+When familiarizing yourself with your robot's operation, always hold the left button (L1/LB).  Once you are comfortable
+with how it operates and you are in a large area with plenty of open room, then you can use R1/RB to enable
+turbo mode.
+:::
+
+##### Differential Drive Robots (Husky, Jackal, Dingo-D, Warthog)
+
+To drive your differential drive robot, Axis 0 controls the robot's steering, Axis 1 controls the
```

Comment:
Consider adding emphasis to 

- Axis 0
- Axis 1
- buttons 4
- 5

Reply:
Done